package.json

```diff
@@ -5,14 +5,18 @@
   "scripts": {
     "build": "webpack -c ./webpack.config.ts --env production",
     "dev": "webpack -w -c ./webpack.config.ts --env development",
+    "e2e": "playwright test -x --trace on",
```

Comment:
To improve the DevX, we should add the [--ui flag](https://playwright.dev/docs/test-ui-mode)

Reply:
Nice 👍